tutorials/multi-tenant-tutorial.rst

```diff
@@ -0,0 +1,166 @@
+.. _multi_tenant_tutorial:
+.. highlight:: sql
+
+Multi-tenant data model
+########################
+
+In this tutorial, we will use a sample ad analytics dataset to demonstrate how you can       
+use Citus to power your multi-tenant application.                             
+
+.. note::
+                                                                                             
+    This tutorial assumes that you already have Citus installed and running. If you don't have Citus running,
+    you can provision a cluster using `Citus Cloud <https://console.citusdata.com>`_ or setup Citus locally
+    using :ref:`single_machine_docker`.
```

Comment:
Unrelated question on the Docker install page.

@jasonmp85, @begriffs -- I had quick questions on a few discrepancies between our Docker setup installations on our docs and Github readme. @sumedhpathak -- I'm planning to open an issue on this, does including the following in that issue work?

* We point to Docker Compose in our Readme. We give the user the option to install Docker Engine or Docker Compose in our docs. Which one should we recommend?
* We ask the user to pick docker-compose for Linux or Mac and also ask them to run `eval`. We don't do that in our docs. Why is that?
* We ask the user to shut down Docker at the end. If they do and then come to this page, do we break the flow?